src/wrapper.js

```diff
@@ -35,10 +36,16 @@
 	}
 
 // Pass this if window is not defined yet
-}(typeof window !== "undefined" ? window : this, function( window, noGlobal ) {
+}( typeof window !== "undefined" ? window : this, function( window, noGlobal ) {
 
 // Support: Firefox 18+
 // Can't be in strict mode, several libs including ASP.NET trace
 // the stack via arguments.caller.callee and Firefox dies if
 // you try to trace through "use strict" call chains. (#13335)
 //"use strict";
+
+// @CODE
+// build.js inserts compiled jQuery here
```

Comment:
Could we get this comment in the same line as `@CODE` so that it disappears in the built file? It's weird to stil have this comment in the full file.